### PR TITLE
Recreated Script 4

### PR DESCRIPTION
**Pull request by nphaterp**
Hey guys. I am almost done script 4. 

I would appreciate feedback on the following: 

- The pvalue from analysis is very small. Right now the table is showing that the pval is 0. Does anyone know how we can fix this ?

### CODE REVIEW

**Comment by nphaterp** [comment]
README.rmd is still there, but I noticed that there were some small edits that we made to the README.md that we didnt make the the .rmd file. I have made those changes now.

**Comment by nichowu** [comment]
Just a reminder as discussed in the meeting, please remove the part that saves the summary statistic table as .png file in script4 because it will be done in script#3. Thanks!

**Comment by nphaterp** [comment]
Removed now

**Comment by nphaterp** [comment]
The function has been removed.

**Comment by fatse** [comment]
@nphaterp can you please look into the comments I left in this PR https://github.com/UBC-MDS/covid-19-cases-vs-tests-analysis/pull/39? 

They pertain to the script you're submitting in this PR.

**Comment by nphaterp** [comment]
I think I am still using the sum_data when I calculated t-stat other than the sum_data save function.

**Comment by fatse** [comment]
But we aren't outputting the `.csv` with stat summary anymore. @nichowu I guess in script 3 we should also save the stat summary table as `.csv` in addition to saving it as `.png`.

File: src/script4.R [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  try({dir.create(out_dir)})
+  bartlett_test_save(data, out_dir)
+  ratio_bootstraps <- ratio_bootstraps(data)
+  test_stat <- diff(sum_data$median_response_ratio)
+  ci_threshold <- quantile(ratio_bootstraps$stat, c(0.025, 0.975))
+  null_dist_save(ratio_bootstraps, ci_threshold, test_stat, out_dir)
+  p_val_save(ratio_bootstraps, test_stat, out_dir)
+}
+
+#' Performs a Bartlette Test to test for equal variances
```
Comment (tanmaysharma19) [inline src/script4.R:38]: I think it's spelled as "Bartlett's test."

**Comment by nphaterp** [comment]
Merged the script-4 clean PR instead

**Pull request closed without merging**